docs/topics/forms/formsets.txt

```diff
@@ -415,6 +415,11 @@ deletion, is greater than or equal to ``min_num``.
     >>> formset.non_form_errors()
     ['Please submit 3 or more forms.']
 
+.. note::
+
+    When a formset contains no initial data, the number of forms displayed is
+    the sum of ``min_num`` and ``extra``.
```

Comment:
Maybe:
```suggestion
    Regardless of ``validate_min``, if a formset contains no data, then
    ``extra + min_num`` empty forms will be displayed.
```

Reply:
I've spent some time thinking about your suggestion. In short, I can't think of anything better. 

I didn't originally include a note about `validate_min` this is about `min_num`; however, as it's in a section about validation I think the clarification is helpful. 

I then thought about how to reword it, but the options I thought of resulted in another full sentence to explain the `validate_min` point which you have refined to just two words.